### PR TITLE
logout test case

### PR DESCRIPTION
**Pull request by skp301565**
This is new test case for logout

### CODE REVIEW

**Comment by demosandygit** [comment]
ok

**Comment by skp301565** [comment]
done

File: zevene_walken/src/test/java/com/ebanking/zevene_walken/Logout.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.Test;
+
+public class Logout
+{
+	@Test
+	public void logout()
+	{
+		System.setProperty("webdriver.chrome.driver", "./drivers/chromedriver.exe");
+		WebDriver driver = new ChromeDriver();
```
Comment (demosandygit) [inline zevene_walken/src/test/java/com/ebanking/zevene_walken/Logout.java:14]: done

**Review by demosandygit — APPROVED** [review]
LGTM

**Pull request merged**